src/express-cli.js

```diff
@@ -8,6 +8,7 @@ import { monitor } from "./express/commands/monitor";
 import { restartAll, restartBor, restartHeimdall } from "./express/commands/restart";
```

Comment:
Would you please merge master in your own branch?
That gives you the opportunity to run `prettier` and `linter` and fix eventual issues to avoid warnings/errors during CI run.
Thank you

Reply:
Done ✅